index.html

```diff
@@ -2905,6 +2915,152 @@ <h3>Evidence</h3>
 
       </section>
 
+      <section>
+        <h3>Confirmation Method</h3>
+
+        <p>
+Confirmation Method can be included by an <a>issuer</a> in a <a>verifiable
+credential</a> to declare that the <a>subject</a> controls one or more
```

Comment:
not sure if `controls` is the right term here. perhaps "has access to" or "is capable of using"

Reply:
Perhaps "controls or has been delegated the ability to use".

Reply:
I'm ok with "controls or has been designated the ability to use" and I updated the PR.  @dlongley @decentralgabe does this work for you?

Reply:
yes!